content/guides/core/automations/_index.md

```diff
@@ -6,4 +6,36 @@ menu:
 title: Automations
 weight: 4
 ---
+{{% pageinfo color="info" %}}
+{{< readfile file="/_includes/enterprise-only.md" >}}
+{{% /pageinfo %}}
 
+This page describes _automations_ in W&B. [Create an automation]({{< relref "create-automations/" >}}) to trigger workflow steps, such as automated model testing and deployment, based on an event in W&B, such as when an [artifact]({{< relref "/guides/core/artifacts" >}}) artifact version is created ({{< relref "/guides/core/registry/" >}}).
+
+For example, an automation can post to a Slack channel when a new version is created, or can run a webhook to trigger automated testing when the `production` alias is added to an artifact.
+
+## Overview
+An automation can run when a specific [event]({{< relref "automation-events.md" >}}) occurs in a registry or project.
+
+For an artifact in a [Registry]({{< relref "/guides/core/registry/">}}), you can configure an automation to run:
+- When a new artifact version is linked to a collection. For example, trigger testing and validation workflows for new candidate models.
+- When an alias is added to an artifact version. For example, trigger a deployment workflow when the `deploy` alias is added to a model version.
```

Comment:
Unsure how familiar users are with aliases compared to tags (and what they are used for). I'd omit this to prevent confusion. 

```suggestion
- When an alias is added to an artifact version. For example, trigger a deployment workflow when an alias is added to a model version.
```